### ISSUE TITLE
The percentages not computed correctly for award ceremony.

### ISSUE THREAD

**Issue by lajp** [Testausserveri/testauskoira-rs#27]
Once again, this should be an easy fix.
![image](https://user-images.githubusercontent.com/31856315/149032186-a806e2da-3e36-4152-8697-29c824e72d1f.png)

**Comment by Esinko** [comment]
Any idea why this issue is still open? This bug is visible every day...
Also, is it possible to add some color to that embed? Perhaps `#445282`?

**Comment by lajp** [comment]
Yes I'll close this an make another issue about having some color in the embed.
